src/imap_db/migrations/env.py

```diff
@@ -1,10 +1,9 @@
 from logging.config import fileConfig
 
 from alembic import context
+from imap_db.model import Base
```

Comment:
Maybe we should configure the import sorter to move our APIs in a separate block at the end. I preferred them being separate. But I can do that!

Reply:
That would be nice!